app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -166,78 +168,57 @@ private fun HelpCenterHomeScreen(
           )
         }
         Spacer(Modifier.height(40.dp))
-        HelpCenterSection(
-          title = stringResource(id = R.string.HC_QUICK_ACTIONS_TITLE),
-          chipContainerColor = MaterialTheme.colorScheme.typeContainer,
-          contentColor = MaterialTheme.colorScheme.onTypeContainer,
-          content = {
-            AnimatedVisibility(
-              visible = quickActions.isNotEmpty(),
-              enter = fadeIn(),
-              exit = fadeOut(),
-            ) {
-              HedvigGrid(
-                insideGridSpace = InsideGridSpace.Companion.invoke(8.dp),
-                modifier = Modifier.padding(horizontal = 16.dp),
-              ) {
-                for (quickAction in quickActions) {
-                  HedvigCard(
-                    onClick = {
-                      onQuickActionsSelected(quickAction)
-                    },
-                    modifier = Modifier
-                      .fillMaxWidth()
-                      .windowInsetsPadding(WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal)),
-                  ) {
-                    Column(
-                      verticalArrangement = Arrangement.Center,
-                      horizontalAlignment = Alignment.CenterHorizontally,
-                    ) {
-                      Text(
-                        text = stringResource(quickAction.titleRes),
-                        textAlign = TextAlign.Center,
-                        modifier = Modifier.padding(16.dp),
-                      )
-                    }
-                  }
-                }
-              }
-            }
-            AnimatedVisibility(
-              visible = commonClaims.isNotEmpty(),
-              enter = fadeIn(),
-              exit = fadeOut(),
-            ) {
+        AnimatedVisibility(
+          visible = quickLinks.isNotEmpty(),
+          enter = fadeIn() + expandVertically(clip = false, expandFrom = Alignment.CenterVertically),
+          exit = fadeOut() + shrinkVertically(clip = false, shrinkTowards = Alignment.CenterVertically),
+        ) {
+          HelpCenterSection(
+            title = stringResource(R.string.HC_QUICK_ACTIONS_TITLE),
+            chipContainerColor = MaterialTheme.colorScheme.typeContainer,
+            contentColor = MaterialTheme.colorScheme.onTypeContainer,
+            content = {
               HedvigGrid(
-                insideGridSpace = InsideGridSpace.Companion.invoke(8.dp),
+                contentPadding = WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal).asPaddingValues(),
```

Comment:
The HedvigGrid layout itself is pretty complicated:) I guess I have one question here about WindowInsets here, I am not very familiar with them. Why do we apply WindowInsets.safeDrawing as contentPadding to the HedvigGrid, StillNeedHelpSection separately here, and not apply Modifier.windowInsetsPadding(WindowInsets.safeDrawing) to the whole Surface/Column on top? Wouldn't it do the same thing? I see that you got rid of Modifier.windowInsetsPadding(WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal)) for individual HedvigCards a bit lower.

Reply:
Yes, applying them to the entire layout would indeed be the same thing as applying it to each item of the layout individually in many cases.
Now why do we need to do it on individual items in some layouts?

In this one in particular, if we did that, then we would not allow the `StillNeedHelpSection` would not actually be able to go edge to edge. Since it is also part of the outer column, if the column already had those insets, then the full area that the `StillNeedHelpSection` would be able to take would be everything *except* for the insets horizontally. Insets in this context being basically the navigation bar.

If the column does *not* have the padding on itself:
![image](https://github.com/HedvigInsurance/android/assets/44558292/e2c4a811-f24e-4d53-bdbf-7140a31666b9)

If the column does have the inset padding on itself and none of the children add their own insets:
![image](https://github.com/HedvigInsurance/android/assets/44558292/a4a6fa85-32bb-4300-ad78-8dbbb5422c58)

Notice how on the left, the chat section is cut-off on the left, since that's where the camera lives on the real device (not visible in the screenshot)


----------------

Then for me getting rid of the insets from each card individually:
The thing which we want to move according to the insets is in this case the entire grid all together. So even by simplicity sake, this is in fact easier to read rather than having to go into the cards individually.
But most importantly, let's think of this scenario:
* We are in portrait mode, and on the right side of the app are the three-buttons for navigation, therefore we got some insets on the right side.
* We take those insets and we apply it on every card individually.
* Now since we are on a grid, the cards are laid out in a 2-column way, so we got cards on the left, and cards on the right
* Since we apply the insets on _all_ cards, the card on the right is moved a bit towards the left, to not be under the navigation bar (this is good). But then the card on the left side of the grid _also_ receives the same insets on the right side. So now it has some space on the right for no reason at all (this is the bad part)

So by moving the insets on the entire grid, we make it simpler while also fixing the bug where the insets are applied twice, in two different places while that is not what we want here.

Existing behavior:

![image](https://github.com/HedvigInsurance/android/assets/44558292/bbdefe68-b4f4-43db-a32a-ee0b4bfe61d6)


Behavior after this PR:

![image](https://github.com/HedvigInsurance/android/assets/44558292/f1d927b5-760e-4529-a2a0-15c2cfa5ce7b)

Reply:
Thanks, that's very detailed!